package.json

```
@@ -49,8 +49,9 @@
     "codecov": "^2.3.1",
     "commitizen": "^2.10.1",
     "create-require": "^1.1.1",
+    "cross-env": "^7.0.3",
```

unfortunately we support 10.12, and cross-env 7 requires 10.14. we can use v6 tho.
```suggestion
    "cross-env": "^6.0.3",
```